code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -633,6 +633,10 @@
 		current_charges--
 		if(recharge_rate)
 			START_PROCESSING(SSobj, src)
+		if(istype(hitby, /obj/item/projectile))
+			var/obj/item/projectile/P = hitby
+			if(P.shield_buster)
+				current_charges = (max(0, (current_charges -= 3)))
```

Comment:
```suggestion
				current_charges = max(0, current_charges - 3)
```
`-=` will assign it already. Which doesn't do anything here. Also the parentheses are not needed

Reply:
Done